addon/components/pikaday-input.js

```diff
@@ -36,6 +36,10 @@ export default Component.extend(PikadayMixin, {
       this.set('value', null);
       this.get('onSelection')(null);
     }
+    
+    if (this.get('clearInvalidDate')) {
+      this.verifyInvalidDate(this.$());
```

Comment:
If we need to pass the element in as an argument, can we pass a reference to `this.element` rather than `this.$()` to decouple from `jQuery`?